bedrock/mozorg/templates/mozorg/moss/index.html

```diff
@@ -10,12 +10,6 @@
   {{ _('Mozilla Open Source Support') }}
 {% endblock %}
 
-{% block header_image %}
-  <div id="header-image-wrapper">
-    <img src="{{ static('img/mozorg/moss/header-home.jpg') }}" alt="">
```

Comment:
We should remove this image file as well, as it's no longer used anywhere.

(The image is located here: `/media/img/mozorg/moss/header-home.jpg`)